modules/openapi-generator/src/main/resources/php/api.mustache

```diff
@@ -54,19 +54,47 @@ use {{invokerPackage}}\ObjectSerializer;
      */
     protected $headerSelector;
 
+    /**
+     * @var Host index
```

Comment:
A 1st keyword of `@var` identifies the type of the property. so I think it would be `@var int Host index` .

Reply:
👍  I'll update it in the next push